lib/cache.js

```diff
@@ -29,15 +40,17 @@ export default {
   },
 
   set: (key, data) => {
-    if (_.isNull(client)) return false;
+    if (isNull(client)) return false;
 
     const timestamp = new Date().getTime();
-    if (_.isArray(data)) {
+    if (isArray(data)) {
       data.map(datum => datum.cached = timestamp); // eslint-disable-line no-param-reassign
     } else {
       data.cached = timestamp; // eslint-disable-line no-param-reassign
     }
 
-    return client.set(key, JSON.stringify(data));
+    return client.set(key, JSON.stringify(data), err => {
```

Comment:
Actually I hadda switch libs cuz of Memcachier's reliance on the binary protocol; this one is less annoying. But just passing along the error for sanity's sake